frontends/election-manager/src/hooks/use_election_manager_store.ts

```diff
@@ -127,7 +109,12 @@ export function useElectionManagerStore(): ElectionManagerStore {
     await queryClient.invalidateQueries(
       getCurrentElectionMetadataResultsQueryKey()
     );
-    await queryClient.invalidateQueries(getCvrFilesQueryKey());
+    await queryClient.invalidateQueries(getCvrFilesQueryKey(), {
+      // TODO(kofi): This is a pattern we're potentially adopting across the
+      // repo and will be make sweeping updates for soon. This specific case is
+      // needed to enable a test to pass after a recent change.
+      refetchType: 'all',
```

Comment:
added this to fix a test that removes the "current" election and then navigates to the "Tally" screen to verify that the previously loaded CVR files are gone.

that particular screen tries to map data in the CVRs to data in the current election, which are out of sync when the screen is loaded and that was causing an exception. very unlikely in practice, but it does surface a potential edge case we should handle in general.

Reply:
Sounds good, glad you surfaced it!